api/modules/crudRefactorClean.js

```diff
@@ -0,0 +1,316 @@
+const sequelize = require('../models');
+
+const { models } = sequelize;
+const { 
+    User,
+    Course,
+    Topic, 
+    Language,
+    Feedback 
+} = models;
+
+
+module.exports = (function () {
+
+    //COURSES
+    const fetchCoursesById = (params) => {
+        const { courseId } = params 
+        return (
+            Course.findAll({
+                where: {
+                    id: courseId
+                }
+            })
+            .then(courses => {
+                console.log('All Courses:', JSON.stringify(courses, null, 4));
+                return courses
+            })
+        )
+    }
+    const fetchCourses = (params) => {
+        return (
+            Course.findAll({
+            })
+            .then(courses => {
+                console.log('All Courses:', JSON.stringify(courses, null, 4));
+                return courses
+            })
+        )
+    }
+    const createCourse = (params) => {
+        const {
+            title,
+            description,
+            current_version,
+            youtube_link,
+            language_id,
+        } = params.values;
+
+        Course.create(
+            {
+                title,
+                description,
+                current_version,
+                youtube_link,
+                language_id,
+            }
+        )
+        .then(course => {
+            console.log('Id of new course:', course.id);
+        });
+
+    } 
+    const updateCourses = (params) => {
+        const { courseId, updatedFields } = params.values
+        
+        Course.update(
```

Comment:
wrap in a return statement (this will be important when trying to call this function from another module)